package.json

```diff
@@ -54,7 +54,7 @@
     "shx": "^0.3.3",
     "tstest": "^0.4.10",
     "wechaty": "^0.56",
-    "wechaty-puppet": "^0.37.1",
+    "wechaty-puppet": "0.37.1",
```

Comment:
Why change this version to be locked?

Reply:
**log**
```log
> wechaty-puppet-service@0.17.2 lint:ts /home/runner/work/wechaty-puppet-service/wechaty-puppet-service
> tsc --noEmit

Error: tests/performance.spec.ts(37,7): error TS2515: Non-abstract class 'PuppetTest' does not implement inherited abstract member 'conversationRead' from class 'PuppetMock'.
```
I do not know how to fix this problem...

Reply:
You have to add `conversationRead` method to the class called `PuppetTest`. This class should be in the test folder.

Reply:
After `conversationRead` been added, it could pass the test, but it is strange that only two methods in class `PuppetTest`.